### PR TITLE
Login endpoint docs

### PR DESCRIPTION
**Pull request by NotFenixio**

### CODE REVIEW

File: docs/cloudlink/commands/authpswd.md [hunk 1]
```diff
@@ -5,7 +5,11 @@ sidebar_label: authpswd
 
 # `authpswd`
 
-This command is used to log into Meower.
+This command is used to log into the Meower CloudLink server.
+
+:::warning
+Although this can be used to get a token, a new and more efficient way has been developed, using a single HTTP request. Head over to the [Authentication section](/rest-api/intro/authentication) to check it.
```
Comment (mybearworld) [inline docs/cloudlink/commands/authpswd.md:11]: `authpswd` is still needed if you want to recieve updates to chats you're in, for example

File: docs/rest-api/intro/authentication.md [hunk 2]
```diff
@@ -17,3 +17,29 @@ If you do not provide a token to endpoints that require authorization, the respo
...
+If any error occurs, including account not existing or incorrect password, a 401 will be returned with the following response:
+```json
+{
+  "error": true,
+  "type": "Unauthorized"
+}
+```
+:::
+
+After that, you will get a [User object](/objects/user).
```
Comment (mybearworld) [inline docs/rest-api/intro/authentication.md]: Don't think that's right

File: docs/cloudlink/commands/authpswd.md (hunk 1 above)
Comment (NotFenixio) [reply docs/cloudlink/commands/authpswd.md:11]: "This command is used to log into the Meower CloudLink server."
That's why I changed that, but yeah, you're right.

File: docs/rest-api/intro/authentication.md (hunk 2 above)
Comment (NotFenixio) [reply docs/rest-api/intro/authentication.md]: Yeah, actually
```json
{
  "account": "user object"
  "error": false,
  "token": "token"
}
```

**Comment by NotFenixio** [comment]
Write the docs yourself!!!!

**Pull request closed without merging**